include/contractor/contractor_config.hpp

```diff
@@ -65,6 +65,7 @@ struct ContractorConfig final : storage::IOConfig
 
     unsigned requested_num_threads;
 
+    // Deprecated
```

Comment:
Please could you add also a release version when it is planed to remove, like `// DEPRECATED to be removed in v6.0`

Reply:
I'll open a follow up issue to track removal in v6 too